oak_runtime/src/lib.rs

```diff
@@ -1254,6 +1299,7 @@ impl Runtime {
         node_name: &str,
         label: &Label,
         initial_handle: oak_abi::Handle,
+        use_privilege: OperationPrivilege,
```

Comment:
Not sure I follow what this means in this context, could you expand the comment?

Reply:
Added a comment.